src/promptflow-core/pyproject.toml

```diff
@@ -43,7 +43,7 @@ packages = [
 python = "<4.0,>=3.8"
 promptflow-tracing = ">=1.0.0,<2.0.0"
 "ruamel.yaml" = ">=0.17.10,<1.0.0"  # used to generate connection templates with preserved comments
-docutils = "*"  # used to generate description for tools
+docutils = "<=0.21"  # used to generate description for tools, temporary limitation to 0.21
```

Comment:
Is this breaks for this version or version above? 

Could we make new version compatible? We might not want to introduce such upper range limit

Reply:
Good point